js/apps/system/_admin/aardvark/APP/react/src/views/services/ServicesReactView.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { ServicesViewWrap } from "./ServicesViewWrap";
+
+const ServicesReactView = () => {
+  return <ServicesViewWrap />;
+};
```

Comment:
Wrapper required?

Reply:
yes, a JSX wrapper is required to avoid some TS issues; can be fixed, but that's the pattern we follow everywhere

Reply:
Yes at some point, but currently it's `require`d in `App.js`, might need to deal with webpack too somewhere